README.md

```diff
@@ -101,6 +101,14 @@ Of course these two items could be combined into the following:
 let g:EditorConfig_exclude_patterns = ['fugitive://.*', 'scp://.*']
 ```
 
+#### Disable for a specific filetype
+
+To disable this plugin when drafting a git commit message:
```

Comment:
Suggest changing to:

> Since you can disable this plugin for a specific buffer, you can disable the plugin for all buffer of a specific filetype.  For example, to disable EditorConfig for all git commit messages (filetype `gitcommit`):

That way it is more clear that this isn't just for a specific filetype.